src/NuGet.Configuration/PackageSource/NuGetConstants.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace NuGet.Configuration
+{
+    public static class NuGetConstants
+    {
+        public static readonly string V3FeedUrl = "https://api.nuget.org/v3/index.json/";
```

Comment:
There shouldn't be a trailing slash on the v3 feed URL.  It'll actually fail with the trailing slash.